OpenFn/kit
# Next release

This branch contains a bunch of fixes for the next set of releases.

I am suspicious of the adaptor-docs build here. It seems to work fine, but I've removed a bunch of build stuff to get it working on the web in the first place. I mean I love deleting code, and I love it when stuff works. But it took a while to get this working in the first place and it's a bit suss that I can just delete it...

Suggest we release from here then merge into main.


## Changes
This PR includes the following new releases:

* adaptor-docs 0.0.5
  * Collapsible sections
  * Show loading when the specifier prop changes
* describe-package 0.0.10
  * Don't describe beta
  * Only describe public functions
  * Include exports from language-common
* CLI 0.0.17
  * Use latest compiler dependency
* Compiler 0.0.17
  * updated dependency on describe package
  * Extended globals list

## QA

I would recommend a little bit of QA

* Build locally
* Run tests locally
* Install the CLI globally (`pnpm pack:local` and `npm install -g dist/openfn-cli-0.0.17``and run `openfn test` to ensure it's working
* Load adaptor docs into Lightning (`pnpm export describe-package && pnpm export adaptor-docs`) - docs should load collapsed

## Related PRs

Change tracking:

* [x] Build stuff #95 
* [x] Fix build #98
* [x] Hide beta from docs #97
* [x] Collapsible docs #94

When this is released we can action #96

Also: we need a better release process. After this, I'll start creating `release/` branches with sets of related releases on them. Should make things easier to organise.
